tests/scripts/power-macro

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+mkdir "$PROG_HOME/tests/power-macro/out"
+
+dotc -d "$PROG_HOME/tests/power-macro/out" "$PROG_HOME/tests/PowerMacro.scala"
+
+measure "$PROG_HOME/tests/PowerInlined-1.scala" -classpath "$PROG_HOME/tests/power-macro/out"
+measure "$PROG_HOME/tests/PowerInlined-1k.scala" -classpath "$PROG_HOME/tests/power-macro/out"
```

Comment:
You also need to add a chart in [bin/charts.js](https://github.com/liufengyun/bench/blob/master/bin/charts.js)